### PR TITLE
Fix CI

### LINKED ISSUE #491: master CI broken

### PR DESCRIPTION
**Pull request by marek1840**
closes #491

### CODE REVIEW

File: tests/com/twitter/intellij/pants/integration/TargetFileCompletionIntegrationTest.java [hunk 1]
```diff
@@ -90,6 +93,10 @@ private void completionTest(String stringToComplete, String[] expected) {
     String fullStringToComplete = "\n\n" + stringToComplete;
     // should be only tested with pants versions above 1.24.0
     if (PantsUtil.isCompatiblePantsVersion(myProjectRoot.getPath(), "1.24.0")) {
+      // invalidate caches
```
Comment (tpasternak) [inline tests/com/twitter/intellij/pants/integration/TargetFileCompletionIntegrationTest.java]: The method name is actually self-descripting :)

File: tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java [hunk 2]
```diff
@@ -20,48 +20,41 @@
...
-    PsiClass testClass = JavaPsiFacade.getInstance(myProject).findClass(classReference, GlobalSearchScope.allScope(myProject));
-    assertNotNull(testClass);
+    PsiClass testClass = ProjectTestJvms.anyTestClass(myProject, getProjectPath());
 
     ExternalSystemRunConfiguration esc = getExternalSystemRunConfiguration(testClass);
 
     // Make sure task name is `test` goal.
-    assertEquals(Collections.singletonList("test"), esc.getSettings().getTaskNames());
+    String testTask = "test";
+    assertEquals(Collections.singletonList(testTask), esc.getSettings().getTaskNames());
```
Comment (wisechengyi) [inline tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java]: can inline 
```
Collections.singletonList("test")
```

File: tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java [hunk 3]
```diff
@@ -20,48 +20,41 @@
...
-
-    List<String> expectedConfigScriptParameters = Arrays.asList(
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight-test-platform",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:six",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:seven",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:base",
-      "--test-junit-test=" + classReference
-    );
-    assertEquals(expectedConfigScriptParameters, configScriptParameters);
```
Comment (wisechengyi) [inline tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java]: Do you think it'd be possible to wrap this around


```
if (PantsUtil.isCompatiblePantsVersion(getProjectPath(), "1.25.0")) { 
...
} 
else {
// testprojects/tests/java/org/pantsbuild/testproject/testjvms:base
// testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight
// testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight-test-platform
}

```

File: tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java [hunk 4]
```diff
@@ -76,34 +69,21 @@ public void testClassRunConfiguration() throws Throwable {
...
-      "--no-test-junit-timeouts",
-      "--jvm-test-junit-options=" + debuggerSetup,
-      "test",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight-test-platform",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:six",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:seven",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:eight",
-      "testprojects/tests/java/org/pantsbuild/testproject/testjvms:base",
-      "--test-junit-test=" + classReference
-    );
```
Comment (wisechengyi) [inline tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java]: same here

File: tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java (hunk 3 above)
Comment (marek1840) [reply tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java]: Sorry, I don't think I follow. 
The highlighted code is removed from repository as it is no longer necessary - now, we don't use hardcoded names of the files but rather inspect the directory and get expected names from there. This should work the same way regardless of the pants version.

Comment (wisechengyi) [reply tests/com/twitter/intellij/pants/execution/OSSPantsJvmRunConfigurationIntegrationTest.java]: I see. I guess that works, too. Thanks!

My original thought was to keep the existing mechanism, and assert on different targets based on the pants version in the pantsbuild/pants repo.

**Pull request merged**